setup.py

```diff
@@ -12,5 +12,8 @@
         'pyledger @ https://github.com/macxred/pyledger/tarball/main',
         'cashctrl_api @ https://github.com/macxred/cashctrl_api/tarball/main'
     ],
-    packages=find_packages(exclude=('tests', 'examples'))
+    packages=find_packages(exclude=('tests', 'examples', 'scripts')),
+    scripts=[
+        'scripts/reset_default.py',
```

Comment:
The name `reset_default.py` is weak and somewhat redundant. 
Maybe think of using this script as command in the terminal to find better names.

Alternative suggestions: `restore_cashctrl.py`,  `restore_original_settings.py`, `restore_cashctrl_to_factory_settings.py`, `restore_initial_state.py`, `retore_initial_cashctrl_state.py`, etc.

Btw: When using argparse in a pyhton script to mimic any other command line tool with `--help` and parameters,  I usually drop the .py ending in order for the script to feel more like a command line tool.